### PR TITLE
RBAC: Add concourse-deployments ClusterRole

### PR DESCRIPTION
**Pull request by r4vi**
And bind it to the concourse-main default service account so that concourse can
report on rollout status.

https://trello.com/c/ofx66AHU/315-show-deployment-rollout-status-for-concourse-app-builds

### CODE REVIEW

File: charts/rbac/templates/concourse-deployments.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+metadata:
+  name: concourse-deployments
+  labels:
+    chart: {{ template "rbac.chart" . }}
+  namespace: {{ .Values.teams.concourseDeploymentStatus.appsNamespace }}
+  annotations:
+    description: {{ .Values.teams.concourseDeploymentStatus.description | quote }}
+rules:
+  - apiGroups:
+      - "extensions"
```
Comment (djsd123) [inline charts/rbac/templates/concourse-deployments.yaml]: Might want to add `apps` too

Comment (r4vi) [reply charts/rbac/templates/concourse-deployments.yaml]: @djsd123 it doesn't need it, so just adding the minimum - what's the reason for adding?

**Comment by djsd123** [comment]
@r4vi We usually add a row to the README table

File: charts/rbac/templates/concourse-deployments.yaml (hunk 1 above)
Comment (djsd123) [reply charts/rbac/templates/concourse-deployments.yaml]: Just thinking if concourse was ever upgraded in future.  Not a problem though

**Comment by r4vi** [comment]
@djsd123 updated readme and changelog :scroll: good spot

**Comment by djsd123** [comment]
LGTM

File: charts/rbac/templates/concourse-deployments.yaml (hunk 1 above)
Comment (r4vi) [reply charts/rbac/templates/concourse-deployments.yaml]: I see what you mean now, `deployments` will move to the `apps` api group eventually so good to do it now so things don't break when it does. I'll make the change

**Pull request merged**